config_scheduler.go

```diff
@@ -108,6 +110,15 @@ func (c *SchedulerConfig) ValidateAndDefault() error {
 			ValidHostAllocatorRoundingRules, c.HostAllocatorRoundingRule)
 	}
 
+	if c.HostAllocatorFeedbackRule == "" {
+		c.HostAllocatorFeedbackRule = HostAllocatorNoFeedback
```

Comment:
So here we default to HostAllocatorNoFeedback, but there _is_ a HostAllocatorDefaultFeedback option which is the empty string, so I think that doesn't make sense (also because HostAllocatorDefaultFeedback isn't a part of ValidDefaultHostAllocatorFeedbackRules, so the word Default seems to be used in a few different ways?). Maybe HostAllocatorDefaultFeedback should be EmptyFeedback, or the ValidDefaultRules can be ValidDistroRules or etc.

Reply:
This is confusing. I changed HostAllocatorDefaultFeedback -> HostAllocatorUseDefaultFeedback which hopefully makes more sense